desafio6.sql

```diff
@@ -0,0 +1,50 @@
+CREATE VIEW faturamento_atual AS
+SELECT
+  MIN(
+    (
+      SELECT
+        valor
+      FROM
+        SpotifyClone.plano
+      WHERE
+        SpotifyClone.usuario.plano_id = SpotifyClone.plano.id
+    )
+  ) AS faturamento_minimo,
+  MAX(
+    (
+      SELECT
+        valor
+      FROM
+        SpotifyClone.plano
+      WHERE
+        SpotifyClone.usuario.plano_id = SpotifyClone.plano.id
+    )
+  ) AS faturamento_maximo,
+  ROUND(
+    AVG(
+      (
+        SELECT
+          valor
+        FROM
+          SpotifyClone.plano
+        WHERE
+          SpotifyClone.usuario.plano_id = SpotifyClone.plano.id
+      )
+    ),
+    2
+  ) AS faturamento_medio,
+  ROUND(
+    SUM(
+      (
+        SELECT
+          valor
+        FROM
+          SpotifyClone.plano
+        WHERE
+          SpotifyClone.usuario.plano_id = SpotifyClone.plano.id
+      )
+    ),
+    2
+  ) AS faturamento_total
+FROM
+  SpotifyClone.usuario;
```

Comment:
Ficou ótimo o código! Mas você viu que teve que repetir as subqueries 3 vezes porque pegou as mesmas informações da tabela plano? Então acho que aqui daria pra usar o join, dessa forma: 
```suggestion
  SELECT 
    MIN(planos.valor) AS faturamento_minimo,
    MAX(planos.valor) AS faturamento_maximo,
    ROUND(AVG(planos.valor), 2) AS faturamento_medio,
    SUM(planos.valor_plano) AS faturamento_total
FROM
    SpotifyClone.usuario AS usuarios
        LEFT JOIN
    SpotifyClone.plano AS planos ON planos.plano_id = usuarios.plano_id;
```